app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -2,4 +2,5 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@drawable/ic_launcher_background"/>
     <foreground android:drawable="@drawable/ic_launcher_foreground"/>
+    <monochrome android:drawable="@drawable/ic_launcher_monochrome"/>
```

Comment:
Couldn't we just use `ic_launcher_foreground`?

Reply:
> Couldn't we just use `ic_launcher_foreground`?

Unfortunately not, as it only comprises of the center circle, not the line across, unless that's what you want:

![ic_launcher_foreground](https://github.com/tananaev/passport-reader/assets/57289288/9629152c-a678-4710-ab1b-48171e6cfcbd)

Reply:
Doesn't think mean that the icons won't actually match? What does that new icon look like side by side with the old?

Reply:
> Doesn't think mean that the icons won't actually match? What does that new icon look like side by side with the old?

Old vs New vs New (Night Mode) Square (Yellow Theme):
![image](https://github.com/tananaev/passport-reader/assets/57289288/512782cb-26cb-4a26-87c9-7735893cb402)
![image](https://github.com/tananaev/passport-reader/assets/57289288/6ab704f5-61b7-44db-ab78-570134e6d6d2)
![image](https://github.com/tananaev/passport-reader/assets/57289288/1f282004-03ca-46bf-84c8-e7a94d80ab44)

Old vs New vs New (Night Mode) Circle (Yellow Theme):
![image](https://github.com/tananaev/passport-reader/assets/57289288/c7eee081-c1cf-44e0-b327-6bbaf2431880)
![image](https://github.com/tananaev/passport-reader/assets/57289288/63177fbf-6465-459f-a1e7-1ab7a54bd7a7)
![image](https://github.com/tananaev/passport-reader/assets/57289288/994d2278-c7f3-42c4-bde0-6ac64ccc2710)